### ISSUE TITLE
Terminating app due to uncaught exception 'NSInvalidArgumentException', reason: '+[FBSDKTypeUtility URLValue:]: unrecognized selector sent to class 0x1007cec68'

### ISSUE THREAD

**Issue by China-Deng** [googleads/googleads-mobile-unity#1952]
### Step 0: Are you in the right place?

  * For general technical questions, or help with project-specific issues like setting up ads in
   your app, reach out to our support team on the
   [Developer Forum](https://groups.google.com/forum/#!categories/google-admob-ads-sdk/game-engines).
  * For assistance with your AdMob account, reach out to
   [AdMob Support](https://support.google.com/admob/?hl=en#topic=7383088).
  * For feedback on [our documentation](https://developers.google.com/admob/unity/start),
   send  your feedback by pressing the **Send Feedback** button at the top right of the
    documentation page you are on.
  * For issues related to __the code in this repository__, continue filing this GitHub issue.

  * Once you've read this section and determined that your issue is appropriate for
    this repository, **please delete this section**.

### [REQUIRED] Step 1: Describe your environment

  * Unity version: 2021.2.10F1C1
  * Google Mobile Ads Unity plugin version: 6.1.2
  * Platform: iOS 
  * Platform OS version: ios14.2 (eg iOS 10, Android 9)
  * Any specific devices issue occurs on: _____
  * Mediation ad networks used, and their versions: Facebook Audience Network with Mediation

#### Steps to reproduce:

What happened? How can we make the problem occur?
This could be a description, log/console output, etc.

Please share a downloadable sample project that reproduces the bug, if
available. This greatly helps us to diagnose and fix the issue faster.

#### Relevant Code:
[Podfile.lock.log](https://github.com/googleads/googleads-mobile-unity/files/8194184/Podfile.lock.log)
[FaceBookAudienceNetworkCrash.log](https://github.com/googleads/googleads-mobile-unity/files/8194186/FaceBookAudienceNetworkCrash.log)
[Someone here solved the problem](https://github.com/facebook/facebook-ios-sdk/issues/1961#issuecomment-995226500)

 saw a lot of crashes in the background. According to Google, the reason is that the version of audiencenetwork is too low (6.8.0) and FBSDKCoreKit_Basics (= 13.0.0). Can you upgrade your version of FacebookAudienceNetworkMediation  with in time? Or can I upgrade in podfile alone? Will a separate upgrade be incompatible with AdMob?
```
// TODO(you): code here to reproduce the problem
```

**Comment by jeremyburgesspikpok** [comment]
I was having this issue last week, but as of this morning I note that this issue does seem to be fixed with the new version of the Facebook Audience Network Adapter (3.6.1) and the new Google Mobile Ads SDK (7.0.0). 

Unfortunately not all adapters have been updated yet to work with 7.0.0, and the Mediation Test Suite is also currently incompatible.

**Comment by China-Deng** [comment]
> I was having this issue last week, but as of this morning I note that this issue does seem to be fixed with the new version of the Facebook Audience Network Adapter (3.6.1) and the new Google Mobile Ads SDK (7.0.0).
> 
> Unfortunately not all adapters have been updated yet to work with 7.0.0, and the Mediation Test Suite is also currently incompatible.

yes! I upgraded the IOS (not the *. unitypackage file of unity) adapter from pods alone. At present, the crash is slowly decreasing. I haven't checked whether there is a new crash
